content/src/main/content/jcr_root/apps/acs-commons/components/utilities/errorpagehandler/404.jsp

```diff
@@ -26,12 +29,23 @@
         // Check for and handle 404 Requests properly according on Author/Publish
         if (errorPageHandlerService.doHandle404(slingRequest, slingResponse)) {
 
-            final String path = errorPageHandlerService.findErrorPage(slingRequest, resource);
+            SlingHttpServletRequest req = sling.getRequest();
 
+            String resourcePath = StringUtils.removeEnd(resource.getPath(),resource.getName());
+
+            Resource res = resource.getResourceResolver().resolve(req , resourcePath);
+
+
```

Comment:
This seems like logic which should go in the findErrorPage() method. Any particular reason to put it in the JSP?